_data/hosts.yml

```diff
@@ -548,6 +548,22 @@
             semver: 7.1.1    
     last_scanned_at: '2017-01-09T15:57:19+0000'
 -
+    name: 'GalacSYS'
+    url: 'https://www.galacsys.net/hebergement_mutualise_details.html'
+    type: shared
+    default: null
+    versions:
+        44:
```

Comment:
Good god. 

Well, 4.4 is so incredibly old, this website doesn't care to track such things. Can you just omit it from the PR, and we can pretend this didn't happen? 🙈